File: Đề NĐ các năm/20-21/Bai XTB 2021/I91201/TROCHOI.cpp
#include <iostream>
#include<fstream>
using namespace std;

int main()
{
   int n ;
   int a[255];
   ifstream f ("TROCHOI.inp");
   f>>n;
   for (int i=0;i<n;i++)
    f>>a[i];
   ofstream g ("TROCHOI.out");

     f.close();
     g.close();
    return 0;
}

File: Đề NĐ các năm/20-21/THPT GIAO THUY/I67201/trochoi.pas
var fa,fb:text;
    n,i,j,dem:longint;
    s:string;
    x:char;
    b:array[-1..10005] of longint;
    a:array[-1..300,-1..10005] of string;
begin
    assign(fa,'trochoi.inp');reset(fa);
    assign(fb,'trochoi.out');rewrite(fb);
    readln(Fa,n);
    for i:=1 to n do
      begin
        s:='';dem:=0;
        while not eoln(Fa) do
          begin
             read(fa,x);
             if x in ['0'..'9'] then inc(dem);
             s:=s+x;
          end;
        readln(fa);
        inc(b[dem]);
        a[dem,b[dem]]:=s;
      end;
    for i:=0 to 256 do
         for j:=1 to b[i] do
           writeln(fb,a[i,j]);
close(fa);
close(fb);
end.
